src/sensor_state_data/library.py

```diff
@@ -90,3 +90,31 @@ class SensorLibrary:
         device_class=SensorDeviceClass.COUNT,
         native_unit_of_measurement=None,
     )
+    KEG_SIZE__VOLUME_LITERS = BaseSensorDescription(
+        device_class=SensorDeviceClass.KEG_SIZE,
+        native_unit_of_measurement=Units.VOLUME_LITERS,
+    )
+    KEG_TYPE__NONE = BaseSensorDescription(
+        device_class=SensorDeviceClass.KEG_TYPE,
+        native_unit_of_measurement=None,
+    )
```

Comment:
Hopefully we can find something more generic to represent this that will work for other devices in the future.  Will probably mean a breaking change though later on.

Reply:
yeah, no problem to change it later